arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,536 @@
+          "type": "string",
+          "description": "Location of the service"
+        },
+        "sku": {
+          "$ref": "#/definitions/RpSkuTier",
+          "description": "Server SKU"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "description": "Key value pairs"
+          },
+          "description": "tags associated with analysis server"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RpPropertyBag",
+          "description": "Property bag for resource operation"
+        }
+      }
+    },
+    "RpSkuTier": {
+      "description": "Represents the SKU name and tier",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Sku name for frontend requests"
+        },
+        "tier": {
+          "type": "string",
+          "description": "Tier for frontend requests"
+        }
+      }
+    },
+    "RpPropertyBag": {
+      "description": "Property bag",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The current state of the provisioning operation"
+        },
+        "serverFullName": {
+          "type": "string",
+          "description": "Full URI of the Analysis Server"
+        },
+        "asAdministrators": {
+          "$ref": "#/definitions/RpAsAdministrators",
+          "description": "List of users to be made administrators on the server"
+        }
+      }
+    },
+    "RpAsAdministrators": {
+      "description": "Array of administrator user identities",
+      "type": "object",
+      "properties": {
+        "members": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "UPN of the user"
+          },
+          "description": "Collection of administrator user identities"
+        }
+      }
+    },
+    "RpProvisionRequest": {
+      "description": "Provision request specification",
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location of the Analysis server"
+        },
+        "sku": {
+          "$ref": "#/definitions/RpSkuTier",
+          "description": "Sku of the Analysis server"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": { "type": "string" },
+          "description": "Key value pairs of additional properties that can ebe specified"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RpPropertyBag",
+          "description": "Provision request properties"
+        }
+      }
+    },
+    "RpProvisionResult": {
+      "description": "Provision result specification",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
Ideally speaking "RpProvisionResult" should be a child of `"allof": []` "Resource" model; where "Resource" will have properties like id, name, type, location, tags, sku. You can take a look at any other swagger spec: storage, compute, network, etc.